### PR TITLE
Create ko.yml

### PR DESCRIPTION
**Pull request by 24802**
translate korean

### CODE REVIEW

File: locales/ko.yml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    switch-focus: 포커스 전환
+    delete-tile: 타일 지우기
+    pon: 퐁
+    chii: 치
+    minkan: 밍깡
+    ankan: 안깡
+    toggle-red: 적패 전환
+    close: 닫기
+  hand-options:
+    none: None
```
Comment (livewing) [inline locales/ko.yml:43]: Not translated.

File: locales/ko.yml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    chitoitsu: 치또이츠
+    ron-with-bonus: "{{a}} - {{b}}"
+    tsumo-with-bonus-dealer: "{{a}} - {{b}} All"
+    tsumo-with-bonus-non-dealer: "{{a}}/{{b}} - {{c}}/{{d}}"
+    tsumo-dealer: "{{count}} All"
+    tsumo-non-dealer: "{{a}}/{{b}}"
+    tsumo-non-dealer-diff: "{{a}}/{{b}}"
+  settings:
+    rule: 룰
+    empty: Empty
```
Comment (livewing) [inline locales/ko.yml:216]: Not translated.

File: locales/ko.yml [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    save-as: 저장
+    overwrite-save: 덮어쓰기
+    name: 이름
+    untitled-rule: 무제-룰
+    on: On
+    off: Off
```
Comment (livewing) [inline locales/ko.yml:222]: Not translated.

File: locales/ko.yml [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+    language: 언어
+    theme: 테마
+    auto: 자동
+    light: 라이트
+    dark: 다크
+    tile-color: 패 색상
+    auto-inverted: 자동 (반전)
+    score: 점수
+    show-bazoro: 바조로 표시
+    about: About
```
Comment (livewing) [inline locales/ko.yml:251]: Not translated.

File: locales/ko.yml [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+  footer:
+    license: This app is licensed under <1>The MIT License</1>. We assume no liability for this app.
+    credit: The mahjong tile images are from <1>FluffyStuff/riichi-mahjong-tiles</1> (<2>CC BY</2>).
```
Comment (livewing) [inline locales/ko.yml:254]: Not translated.

**Comment by 24802** [comment]
Surprisingly, the parts that are not translated are words that are used more in English than in Korean even in Korea. I decided that it would be more natural to write in English than to forcefully translate it into Korean.

**Review by livewing — APPROVED** [review]
I see. That makes sense.

I'll merge this PR. Thanks!

**Pull request merged**